code/src/index.css

```diff
@@ -1,13 +1,253 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Roboto', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #ef3b36; /* fallback for old browsers */
+  background: -webkit-linear-gradient(
+    to right,
+    #ffffff,
+    #ef3b36
+  ); /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(
+    to right,
+    #ffffff,
+    #ef3b36
+  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+* {
+  padding: 0;
+  margin: 0;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+form {
+  background-color: rgb(233, 231, 231);
+  border: 1px solid black;
+  padding: 1em;
+  width: 324px;
+  margin: 1em auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  box-shadow: 6px 6px 1px 1px rgb(5 5 5 / 90%);
+}
+
+textarea {
+  border: 0;
+  padding-top: 1em;
+  padding-left: 1em;
+  margin: 0;
+  resize: none;
+  min-width: 90%;
+  max-width: 90%;
+  min-height: 1em;
+  font-size: 15px;
+}
+
+.submit-button {
+  background-color: lightpink;
+  font-size: 16px;
+  font-weight: 700;
+  border-radius: 20px;
+  margin: 20px 0 0 7px;
```

Comment:
for some reason button text looks uncentered in Chrome . According to devtolls it will be ok , if you add these lines : `    display: flex;
    justify-content: center;
    align-items: center;`